google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -247,11 +247,29 @@ class Token(_BasePageElement):
             Required. The text of the Token.
         symbols (List[Symbol]):
             Optional. The Symbols contained within the Token.
+        confidence (float):
+            Optional. The confidence score of the Token detection.
+        detected_languages (List[documentai.Document.Page.DetectedLanguage]):
+            Optional. A list of detected languages for this Token.
```

Comment:
Thank you for this contribution; however, can you change this to add these cached properties at the `_BasePageElement` level? All of these elements should be able to access these values.